internal/db/types.go

```diff
@@ -24,8 +24,31 @@ type ChangeRequest struct {
 }
 
 type Service struct {
-	Id         int
-	ResourceId int
+	Id                     int
+	CreatedAt              sql.NullTime
+	UpdatedAt              sql.NullTime
+	Name                   sql.NullString
+	LongDescription        sql.NullString
+	Eligibility            sql.NullString
+	RequiredDocuments      sql.NullString
+	Fee                    sql.NullString
+	ApplicationProcess     sql.NullString
+	ResourceId             sql.NullInt32
+	VerifiedAt             sql.NullTime
+	Email                  sql.NullString
+	Status                 sql.NullInt32
+	Certified              sql.NullBool
```

Comment:
This certified field is also not nullable according to the Rails schema.rb file.

Reply:
Yea I'll do a pass to make sure anything required in DB is expected here.